docs/graphql/manual/guides/integrations/auth0-jwt.rst

```diff
@@ -201,3 +201,8 @@ That’s it! This rule will be triggered on every successful signup/login and sy
 .. note::
 
    We need to use an ``upsert`` operation here because social logins do not distinguish between sign-up and login. Hence, we need to run this rule every time a successful login is made and do nothing if the user already exists.
+
+
+.. admonition:: Local dev with Auth0 rules
+
+   The sync step will require a reachable endpoint to Hasura and this is not possible in localhost. You can use ``ngrok`` or similar services to expose your endpoint with a public endpoint temporarily.
```

Comment:
```suggestion
   When developing locally with Auth0 rules, the sync step will require a reachable endpoint to Hasura which is not possible in localhost. You can use ``ngrok`` or similar services to expose your endpoint with a public endpoint temporarily.
```

Reply:
This is actually ambiguous because you can't develop the *Auth0 rule* locally.

Reply:
Okay then let's keep the bold title and leave this as it was before 🙂